locales/de.yml

```diff
@@ -1,76 +1,76 @@
-lang: German
-hello: Hello NoFWL
+lang: Deutsch
+hello: Hallo NoFWL
 window:
   message:
-    close: Do you want to exit the application when you click the [x] button?
+    close: Willst du die Anwendung schließen, wenn du [x] drückst?
 menu:
   app:
-    about: About NoFWL
-    check_update: Check for Updates
+    about: Über NoFWL
+    check_update: Auf Updates überprüfen
   preferences:
-    name: Preferences
-    control_center: Control Center
-    stay_on_top: Stay On Top
-    titlebar: Titlebar
-    hide_dock_icon: Hide Dock Icon
-    language: Language
+    name: Einstellungen
+    control_center: Kontrollzentrum
+    stay_on_top: Immer oben bleiben
+    titlebar: Titelleiste
+    hide_dock_icon: Taskleistensymbol verstecken
+    language: Sprache
     theme:
-      name: Theme
-      light: Light
-      dark: Dark
+      name: Darstellung
+      light: Hell
+      dark: Dunkel
       system: System
-    sponsor: Sponsor NoFWL
+    sponsor: NoFWL unterstützen
   edit:
-    name: Edit
+    name: Bearbeiten
   view:
-    name: View
-    go_back: Go Back
-    go_forward: Go Forward
-    scroll_top: Scroll to Top of Screen
-    scroll_bottom: Scroll to Bottom of Screen
-    zoom_0: Zoom to Actual Size
-    zoom_out: Zoom Out
-    zoom_in: Zoom In
-    reload: Refresh the Screen
+    name: Fenster
+    go_back: Zurück
+    go_forward: Vorwärts
+    scroll_top: Zum oberen Bildschirmrand scrollen
+    scroll_bottom: Zum unteren Bildschirmrand scrollen
+    zoom_0: Auf Originialgröße zurücksetzen
+    zoom_out: Verkleinern
+    zoom_in: Vergrößern
+    reload: Neu laden
   help:
-    name: Help
-    app_log: NoFWL Log
-    report_bug: Report Bug
+    name: Hilfe
+    app_log: NoFWL-Protokoll
+    report_bug: Bug melden
     feedback: Feedback
-    dev_tools: Toggle Developer Tools
+    dev_tools: Entwicklerwerkzeuge anzeigen/verstecken
```

Comment:
`Entwicklerwerkzeuge umschalten`?

Reply:
I thought about that, but it sounds a bit strange in my opinion